README.md

```diff
@@ -17,17 +17,71 @@ Qualle provides a command-line interface to train
 and evaluate models. In addition, a REST webservice for predicting
 the recall of a MLC result is provided.
 
-### Command line interface (CLI)
-In order to run the CLI, you must install the packages with [poetry](https://python-poetry.org).
-The interface is then accessible from the module ``qualle.main``. To
-see the help message, run (inside the Qualle directory)
+## Installation
 
-``python -m qualle.main -h``
+### Requirements
 
+Python 3.8 or 3.9 is required.
 
-### Train
-In order to train a model you have to provide a training data file.
-This file has to be a tabular-separated file (tsv) in the format (tabular is marked with ``\t``)
+### With pip
+Qualle is available on [PyPI](pypi.org) . You can install Qualle using pip:
+
+``pip install qualle``
+
+This will install a command line tool called `qualle` . You can call `qualle -h` to see the help message which will
```

Comment:
"Qualle" is das Framework, 'qualle' das Tool?

Reply:
Ja. Das Tool heißt wie das Framework. Für das Framework habe ich bereits vorher schon in der Readme Title Case verwendet, und wollte das einheitlich durchziehen.

Reply:
ok